app/graphql/concerns/validate_submission_gradable.rb

```diff
@@ -0,0 +1,38 @@
+module ValidateSubmissionGradable
+  extend ActiveSupport::Concern
+
+  class OwnersShouldBeActive < GraphQL::Schema::Validator
+    def validate(_object, _context, value)
+      submission = TimelineEvent.find_by(id: value[:submission_id])
```

Comment:
From what I can tell, this module is used in grading, adding feedback, and undoing a grading. In all of these circumstances, the presence of the timeline event is mandatory. So this should probably be a `.find`:

```suggestion
      submission = TimelineEvent.find(value[:submission_id])
```